tests/unit/test_jenkins.py

```diff
@@ -5,7 +5,7 @@
 
 # Need access to protected functions for testing
 # All tests belong to single jenkins module
-# pylint:disable=protected-access, too-many-lines
+# pylint:disable=protected-access, C0302
```

Comment:
Similar here, referring to the rule by name makes it easier to know at least which one is disabled without needing to lookup a code 🙂

Reply:
I changed it back to using its name.